packages/ember-routing/lib/system/router.js

```diff
@@ -629,6 +633,7 @@ const EmberRouter = EmberObject.extend(Evented, {
 
     let doUpdateURL = function() {
       location.setURL(lastURL);
+      set(emberRouter, 'currentURL', lastURL);
```

Comment:
I can do that. My thought was I wanted to keep both *set* calls in the same place, but these methods are so closely related that it's find either way.